src/twisted/names/test/test_names.py

```diff
@@ -662,7 +686,56 @@ def test_recordMissing(self):
                 )
             ],
         )
-        self.assertEqual(additional, [])
+        self.assertEqual(resp.additional, [])
+
+    def test_queryNXDOMAINInZone(self):
```

Comment:
Let's at least not add *new* untyped code in here :)

```suggestion
    def test_queryNXDOMAINInZone(self) -> None:
```